index.html

```diff
@@ -8,22 +8,146 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <header class="header-section">
-    <div class="bg-image"></div>
-    <nav>
-        <img src="images/menu.svg" alt="">
-    </nav>
-   </header>
-   <section id="second-main">
-     <h1>AMBROSE <br> ISIGBA</h1>
-     <p>Hello! I am a passionate software developer! my goal
-        is to help you build a quality product(s), features 
-        and/or website with international standard. You are 
-        highly invited to take a tour of my works. If you fall in 
-        love with  my work gallery, and you are planning for one or 
-        you need a code mentor. AMBROSE's CODELAB get you covered</p>
-      <button type="button">Start colaboration</button>
-      <div id="second-section-bg-img"></div>
+  <header id="header-section">
+      <div id="left-float"></div>
+      <nav id="right-float">
+        <a href="#">
+          <img src="images/logo-menu.svg" alt="">
+        </a>
+      </nav>
+  </header>  
+  <main id="section-main">
+        <h1>AMBROSE <br> ISIGBA</h1>
+        <p>Hello! I am a passionate software developer! my goal
+           is to help you build a quality product(s), features 
+           and/or website with international standard. You are 
+           highly invited to take a tour of my works.If you fall in 
+            love with  my work gallery, and you are planning for one or 
+           you need a code mentor. AMBROSE's CODELAB get you covered</p>
+        <button type="button">Start colaboration</button>
+        <div id="second-section-bg-img"></div>
+  </main>
+    <!-- project section -->
+  <section>
+    <div id="project-container">
+      <h2 id="project">Projets</h2>
+    <div id="project-section1" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
```

Comment:
- [ ] Your buttons should be aligned on the same line. please fix it for your design to look similar to the one from figma design. 
![image](https://user-images.githubusercontent.com/43877689/170815319-a0edc0ae-cfef-43e8-a1fe-79e9c04a85c9.png)

Reply:
Done